nautobot_design_builder/tests/testdata/create_or_update_by_ref.yaml

```
@@ -52,12 +52,15 @@ designs:
         "access_type": "Generic"
         "secret_type": "password"
 checks:
-  - count:
-      model: "nautobot.extras.models.Secret"
-      count: 2
-  - count:
-      model: "nautobot.extras.models.SecretsGroup"
-      count: 1
-  - count:
-      model: "nautobot.extras.models.SecretsGroupAssociation"
-      count: 2
+  - equal:
+      - model: "nautobot.extras.models.Secret"
+        count: true
+      - value: 2
```

what about this? (adding the value as a key in the dictionary? It seems more natural
```suggestion
  - equal:
      - model: "nautobot.extras.models.Secret"
        count: true
        value: 2
```

---

Perhaps I am thinking about it wrong, that is kinda the way it was. There was `equal`, `count`, and `count_equal`, which did not seem natural to me. 

`count` was as you described, you were comparing what you sent with a value of. Then there was `count_equal`, which seemed to be something like this

```yaml
  - count_equal:
      - model: "nautobot.extras.models.RelationshipAssociation"
        query: {relationship__label: "Device to VLANS"}
        attribute: "destination"
      - model: "nautobot.ipam.models.VLAN"
```

So, every time I needed count, I was thinking it was `count_equal`. Whereas to me, having 1 way to do comparisons (item 1 in list compared to item 2 in the list) vs multiple (someone times on list, sometimes within dictionary) seemed natural.